src/windowsservercore/ltsc2019/helix/amd64/Dockerfile

```diff
@@ -1,33 +1,69 @@
 # escape=`
 FROM mcr.microsoft.com/windows/servercore:ltsc2019
 
-SHELL ["cmd", "/S", "/C"]
-USER ContainerAdministrator
```

Comment:
user `ContainerAdministrator` must exist for Helix scenarios. this wasn't removed in src/nanoserver/1809/helix/amd64/Dockerfile but was in src/windowsservercore/*

Reply:
That's because in Windows Server Core containers, `ContainerAdministrator` is the default user. But in Nano Server it's `ContainerUser`. That means this line is redundant, which is why I removed it.